sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/EditContactSteps.java

```diff
@@ -98,6 +102,145 @@ public EditContactSteps(WebDriverHelpers webDriverHelpers) {
           softly.assertAll();
         });
 
+    When(
+        "I check the edited data is correctly displayed on Edit Contact page after editing",
+        () -> {
+          aContact = collectContactData();
+
+          SoftAssertions softly = new SoftAssertions();
```

Comment:
Quick question, are we comparing all fields here? Like all POJO's variables? because we can change it with Truth and validate 2 objects